lib/git_reflow/commands/review.rb

```diff
@@ -5,16 +5,45 @@
   c.arg_name '[destination_branch] - the branch you want to merge your feature branch into'
   c.flag [:t, :title], default_value: 'last commit message'
   c.flag [:m, :message], default_value: 'title'
+  c.switch [:e, :edit]
   c.action do |global_options,options,args|
-    review_options = {
-      'base' => args[0],
-      'title' => global_options[:title],
-      'body' => global_options[:message]
-    }
 
-    review_options['title'] ||=  GitReflow.get_first_commit_message
-    review_options['body']  ||=  review_options['title']
+    commit_msg_file = '/tmp/git_reflow_pr_msg'
 
+    if global_options[:title] || global_options[:message]
```

Comment:
since `(global_options[:title] || global_options[:message]) == (editor = false)`, let's just assign this above into a variable: `use_editor = ("#{(global_options[:title] || global_options[:message])}".length == 0)`, then this becomes `unless use_editor` and we can eliminate the other assignments of `editor` below